### DISCUSSION TITLE
Openshift-logging operator -> The field collection.logs.fluentdStatus.pods is invalid

### DISCUSSION THREAD

**Discussion by ImperrattoR** [okd-project/okd#1546] [Q&A] [answered]
Hey,

I have an issue with Openshift cluster-logging operator on OKD. 
Operator version: 5.6.3.

On the cluster logging tab I have created instance:
YAML:
![image](https://user-images.githubusercontent.com/26445096/228470176-86fc545e-5a2f-4208-8aa0-2593f3b2db64.png)

Expected result:
After creating cluster logging instance I should get collection.logs.fluentdStatus.pods inside of the ring.
![image](https://user-images.githubusercontent.com/26445096/228468675-08e2d09b-7005-4228-817c-9c2fb18b1ed2.png)

Actual result:
The field collection.logs.fluentdStatus.pods is invalid.
![image](https://user-images.githubusercontent.com/26445096/228468999-642e7b12-516e-49a6-a1cc-f71b51ce2ea5.png)

![image](https://user-images.githubusercontent.com/26445096/228470282-527f1871-0e95-4503-b28a-d39d3f45f0a8.png)

Does anyone have any idea what could be wrong with my configuration?

**Comment by kr0kar** [comment, marked as answer]
The problem was solved with an operator Red Hat OpenShift Logging upgrade, from 5.6.3 to 5.6.4.
